versions/unversioned/guides/using-apollo.rst

```diff
@@ -0,0 +1,177 @@
+.. _using-apollo:
+
+Using Apollo
+============
+
+The `Apollo <http://www.apollodata.com/>`_ client makes it much easier for you
```

Comment:
I don't think we have to emphasize "much easier" since this is the only section of the docs that talks about GraphQL at all AFAIK.